jmclient/jmclient/cryptoengine.py

```diff
@@ -11,9 +11,9 @@
 # to one wallet type and one engine, not anymore
 #with fidelity bond wallets and watchonly fidelity bond wallet, the wallet class
 # can have two engines, one for single-sig addresses and the other for timelocked addresses
-TYPE_P2PKH, TYPE_P2SH_P2WPKH, TYPE_P2WPKH, TYPE_P2SH_M_N, TYPE_TIMELOCK_P2WSH, \
+TYPE_P2PKH, TYPE_P2SH_P2WPKH, TYPE_P2WPKH, TYPE_P2SH_M_N, TYPE_P2WSH, TYPE_TIMELOCK_P2WSH, \
```

Comment:
changing the order of entries here will break existing wallets

Reply:
Didn't realise that wallets were storing the integer values for the wallet types when created. I have corrected that by adding it to the end.

Reply:
Maybe it's worth adding comment about this here?

Reply:
@kristapsk agreed. I just added a note regarding that.